statefun-python-sdk/README.md

```diff
@@ -1,98 +1,35 @@
-# Apache Flink Stateful Functions
+<img alt="Stateful Functions" src="../docs/fig/stateful_functions_logo.png" width=350px/>
```

Comment:
This README is used as the PyPI project description, and apparently they don't support images.
Would have to use plain titles here.

Reply:
Thanks, Gordon — will remove the images.